xbmc/input/ButtonTranslator.cpp

```diff
@@ -87,7 +87,7 @@ static const ActionMapping actions[] =
     { "showsubtitles"            , ACTION_SHOW_SUBTITLES },
     { "nextsubtitle"             , ACTION_NEXT_SUBTITLE },
     { "cyclesubtitle"            , ACTION_CYCLE_SUBTITLE },
-    { "codecinfo"                , ACTION_SHOW_CODEC },
+    { "playerdebug"              , ACTION_PLAYER_DEBUG },
```

Comment:
While you are at it (changing the cpp and xml): Can't the slide show picture info dialog stuff be changed to react on ACTION_SHOW_INFO which is more appropriate or is this already used differently there?